.github/actions/build-zui/action.yml

```diff
@@ -47,16 +51,45 @@ runs:
         security find-identity -p codesigning -v
       shell: bash
 
+    - name: Checkout esigner-codesign repository
+      if: runner.os == 'Windows'
+      uses: actions/checkout@v3
+      with:
+       repository: 'SSLcom/esigner-codesign'
+       path: esigner-codesign
+
+    - name: Make values from package.json available in Actions steps
+      id: zui-package
+      uses: RadovanPelka/github-action-json@v1.0.1
+      with:
+        path: apps/zui/package.json
```

Comment:
I added this because one of the env variables to be passed to the esigner tool is the expected filename of what would be signed, and that filename includes both the app name (`Zui` or `Zui Insiders`) and the version string, both of which are in `package.json`.